tools/cli/internal/apiversion/version.go

```diff
@@ -24,15 +24,18 @@ import (
 )
 
 type APIVersion struct {
-	version     string
-	versionDate time.Time
+	version          string
+	stabilityVersion string
+	versionDate      time.Time
 }
 
 const (
-	dateFormat = "2006-01-02"
+	dateFormat            = "2006-01-02"
+	StableStabilityLevel  = "STABLE"
+	PreviewStabilityLevel = "PREVIEW"
 )
 
-var ContentPattern = regexp.MustCompile(`application/vnd\.atlas\.(\d{4})-(\d{2})-(\d{2})\+(.+)`)
+var ContentPattern = regexp.MustCompile(`application/vnd\.atlas\.((\d{4})-(\d{2})-(\d{2})|preview)\+(.+)`)
```

Comment:
👀 it's usually a bad idea to export vars because they can be reassigned

Reply:
updated